AMW_business/src/main/java/ch/puzzle/itc/mobiliar/business/apps/boundary/ListAppsUseCase.java

```diff
@@ -0,0 +1,13 @@
+package ch.puzzle.itc.mobiliar.business.apps.boundary;
+
+import ch.puzzle.itc.mobiliar.business.resourcegroup.entity.ResourceWithRelations;
+import ch.puzzle.itc.mobiliar.common.exception.NotFoundException;
+import ch.puzzle.itc.mobiliar.common.util.Tuple;
+
+import java.util.List;
+
+public interface ListAppsUseCase {
+
+    Tuple<List<ResourceWithRelations>, Long> appsFor(Integer startIndex, Integer maxResults, String filter, Integer releaseId) throws NotFoundException;
```

Comment:
We coud create a dedicated object that helps to understand what the Long value stands for - instead of using a Tuple here.